index.js

```diff
@@ -1,63 +1,102 @@
+const request = require('request-promise');
+const SilentError = require('silent-error');
+const DeployPluginBase = require('ember-cli-deploy-plugin');
 
 module.exports = {
   name: 'ember-cli-deploy-sentry',
 
-  contentFor: function(type/*, config*/) {
+  contentFor(type /*, config*/) {
     if (type === 'head-footer') {
       return '<meta name="sentry:revision">';
     }
   },
 
-  createDeployPlugin: function(options) {
-    var DeployPlugin = DeployPluginBase.extend({
+  createDeployPlugin(options) {
+    let DeployPlugin = DeployPluginBase.extend({
       name: options.name,
+      runAfter: ['gzip', 's3'],
+
+      requiredConfig: [
+        'publicUrl',
+        'sentryUrl',
+        'sentryOrganizationSlug',
+        'sentryProjectSlug',
+        'sentryApiKey',
+        'revisionKey'
+      ],
+
       defaultConfig: {
-        distDir: function(context) {
+        filePattern: '/**/*.{js,map}',
+        enableRevisionTagging: true,
+        replaceFiles: true,
+        distDir(context) {
           return context.distDir;
         },
-        filePattern: '/**/*.{js,map}',
-        revisionKey: function(context) {
+        revisionKey(context) {
           return context.revisionData && context.revisionData.revisionKey;
-        },
-        enableRevisionTagging: true,
-        replaceFiles: true
+        }
       },
-      requiredConfig: ['publicUrl', 'sentryUrl', 'sentryOrganizationSlug', 'sentryProjectSlug', 'sentryApiKey', 'revisionKey'],
 
-      prepare: function(context) {
-        var isEnabled = this.readConfig('enableRevisionTagging');
-        if(!isEnabled) {
+      prepare(context) {
+        let isEnabled = this.readConfig('enableRevisionTagging');
+
+        if (!isEnabled) {
           return;
         }
 
-        var revisionKey = this.readConfig('revisionKey');
-        if(!revisionKey) {
-          return new SilentError("Could not find revision key to fingerprint Sentry revision with.");
+        let revisionKey = this.readConfig('revisionKey');
+        if (!revisionKey) {
+          return new SilentError('Could not find revision key to fingerprint Sentry revision with.');
         }
 
         // TODO instead of plainly reading index.html, minimatch
         // getConfig('revision patterns') on context.distFiles
-        var indexPath = path.join(context.distDir, "index.html");
-        var index = fs.readFileSync(indexPath, 'utf8');
-        index = index.replace('<meta name="sentry:revision">',
-                                  '<meta name="sentry:revision" content="'+revisionKey+'">');
+        let indexPath = path.join(context.distDir, 'index.html');
+        let index = fs.readFileSync(indexPath, 'utf8');
+        index = index.replace(
+          '<meta name="sentry:revision">',
+          '<meta name="sentry:revision" content="' + revisionKey + '">'
+        );
         fs.writeFileSync(indexPath, index);
       },
 
-      upload: function(/* context */) {
+      upload(/* context */) {
```

Comment:
This method is all that really changed, focus the review here.  Everything else is formatting and will be discarded when I open up the "official" PR if there is interest.